shells/bash/Portfile

```diff
@@ -23,89 +23,16 @@ master_sites        gnu
 patch_sites         gnu:${name}/${distname}-patches
 dist_subdir         ${name}/${bash_version}_1
 checksums           ${distname}${extract.suffix} \
-                    rmd160  48869b3a460007d05c02ef99745477b2e526fdec \
-                    sha256  d86b3392c1202e8ff5a423b302e6284db7f8f435ea9f39b5b1b20fd3ac36dfcb
+                    rmd160  a081428a896d617855499376b670eca3433a27c1 \
+                    sha256  b4a80f2ac66170b2913efbfb9f2594f1f76c7b1afd11f799e22035d63077fb4d \
+                    size    10135110
 
 # Generate patchfiles
 for {set i 1} {$i <= $bash_patchlevel} {incr i} {
     patchfiles-append \
         [format "%s%s-%03d" $name [strsed ${bash_version} {g/\.//}] $i]
 }
 
-checksums-append    bash44-001 \
-                    rmd160  a4915a389d04165402193ab681ed975b3e8a29d2 \
-                    sha256  3e28d91531752df9a8cb167ad07cc542abaf944de9353fe8c6a535c9f1f17f0f \
```

Comment:
I would like to keep one checksum entry in a comment to make it easier to restore this functionality when new patches for bash 5.0 are released.